bindata/v3.11.0/kube-scheduler/pod.yaml

```diff
@@ -18,7 +18,6 @@ spec:
     volumeMounts:
     - mountPath: /etc/kubernetes/static-pod-resources
       name: resource-dir
-  hostNetwork: true
```

Comment:
This surprises me.  I think the scheduler will remain a hostnetwork pod

Reply:
But previously before this PR, scheduler never needed to run as hostNetwork. What is the scheduler missing by not running as hostNetwork? Who is going to contact scheduler on hostNetwork? Wonder what I am missing.

Reply:
> But previously before this PR, scheduler never needed to run as hostNetwork. What is the scheduler missing by not running as hostNetwork? Who is going to contact scheduler on hostNetwork? Wonder what I am missing.

That just means you were bugged and had a cycle.  If the CVO installs a networking operator, which creates the network, it needs to be scheduled.  Now is a good time to try to fix that.  Especially since the networking team is trying to land.